index.js

```diff
@@ -104,6 +120,10 @@ class SimpleObservable {
   subscribe(handler) {
     handler(this.value);
     this.watchers.push(handler);
+    let unsub = () => {
+      this.watchers.splice(this.watchers.indexOf(handler), 1);
```

Comment:
This is buggy if `unsub` is called multiple times. It'll end up doing `this.watchers.splice(-1, 1)` which removes the last element of the array.

Reply:
Good catch!